.env.libre

```diff
@@ -0,0 +1,89 @@
+# global
+STAGE=dev
+APP_NAME=eosio-dashboard
+
+# wallet
+WALLET_DATA=./wallet_data
+
+# postgres
+POSTGRES_USER=eoscr
+POSTGRES_PASSWORD=password
+POSTGRES_DB=localdb
+POSTGRES_DATA=./db_data
+
+# hasura
+HASURA_GRAPHQL_ENABLE_CONSOLE=true
+HASURA_GRAPHQL_DATABASE_URL=postgres://eoscr:password@postgres:5432/localdb
+HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
+HASURA_GRAPHQL_UNAUTHORIZED_ROLE=guest
+HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
+
+# hapi
+HAPI_EOS_API_NETWORK_NAME=libre
+HAPI_EOS_API_ENDPOINT=https://libre.edenia.cloud
+HAPI_EOS_API_CHAIN_ID=38b1d7815474d0c60683ecbea321d723e83f5da6ae5f1c1f9fecc69d9ba96465
+HAPI_EOS_BASE_ACCOUNT=baseaccount
+HAPI_EOS_BASE_ACCOUNT_PASSWORD=PW...
+HAPI_EOS_MECHANICS_ACCOUNT=eosmechatero
+HAPI_EOS_MECHANICS_PASSWORD=PW...
+HAPI_EOS_WALLET_URL=http://localhost:8888
+HAPI_EOS_BP_JSON_ON_CHAIN=false
+HAPI_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
+HAPI_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
+HAPI_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
+HAPI_HASURA_URL=http://hasura:8080/v1/graphql
+HAPI_HASURA_ADMIN_SECRET=myadminsecretkey
+HAPI_SERVER_PORT=9090
+HAPI_SERVER_ADDRESS=hapi
+HAPI_DATABASE_URL=postgres://eoscr:password@postgres:5432/localdb
+HAPI_SYNC_PRODUCERS_INTERVAL=86400
+HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
+HAPI_SYNC_PRODUCER_CPU_INTERVAL=6
+HAPI_SYNC_EXCHANGE_RATE=86400
+HAPI_EOS_EXCHANGE_RATE_API=https://api.coingecko.com/api/v3/simple/price?ids=proton&vs_currencies=usd
+HAPI_COINGECKO_API_TOKEN_ID=BTC
+HAPI_REWARDS_TOKEN=LIBRE
+HAPI_SYNC_STATS_INTERVAL=60
+HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
+HAPI_CREATE_ACCOUNT_ACTION_NAME=newaccount
+HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+
+#webapp
+PORT=3000
+REACT_APP_TITLE=Libre Network Dashboard
+REACT_APP_VERSION=dev
+REACT_APP_NAME=$npm_package_name
+REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmQ48Sysmfhb9T8Cu3ECNzKCqxTbyuz6n2vWEpTLigmijQ
+REACT_APP_FOOTER_LINKS=[{ "text": "Bitcoin Libre Website", "src": "https://libre.org/" },{ "text": "Libre Block Explorer", "src": "https://libre-explorer.edenia.cloud/" },{"text": "Documentation","src": "https://libre-chain.gitbook.io/"},{"text": "Libre Network Monitor","src": "https://libre.eosio.online"}]
+REACT_APP_EOS_RATE_LINK=
+REACT_APP_USE_REWARDS=false
+REACT_APP_USE_VOTES=false
+REACT_APP_USE_CPU_BENCHMARK=false
+REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
+REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
+REACT_APP_EOS_API_NETWORK_NAME=libre
+REACT_APP_EOS_API_NETWORK_LABEL=Libre Mainnet
+REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmWY6WC6Gy193rKTnHNKzeGruXBGkTQzEVDfwn3j41W235
```

Comment:
no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png